std/experimental/allocator/building_blocks/bitmapped_block.d

```diff
@@ -284,7 +284,8 @@ struct BitmappedBlock(size_t theBlockSize, uint theAlignment = platformAlignment
     Allocations greater than 64 blocks require a multiword search through the
     metadata.
     */
-    @trusted void[] allocate(const size_t s)
+    pure nothrow @trusted @nogc
```

Comment:
Why can this `@trusted`? (Please don't slap `@trusted` on functions.)

Reply:
This was made `@trusted` before. I just added `pure nothrow @nogc`.
Should I make this `@safe` and add `@trusted` lambdas inside?